ql/test/query-tests/InconsistentCode/MissingErrorCheck/tests.go

```diff
@@ -0,0 +1,337 @@
+package test
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strconv"
+)
+
+func mayError(input int) (*int, error) {
+
+	if input%2 == 0 {
+		newp := new(int)
+		*newp = 5
+		return newp, nil
+	} else {
+		return nil, errors.New("oh crap")
+	}
+
+}
+
+func returnsNonNil(input int) (*int, error) {
+
+	newp := new(int)
+	*newp = 5
+
+	if input%2 == 0 {
+		return newp, nil
+	} else {
+		return newp, errors.New("oh crap")
+	}
+
+}
+
+func userDefinedDie() {
+
+	os.Exit(1)
+
+}
+
+func makesCheckUsingSwitch() {
+
+	converted, converr := strconv.Atoi(os.Args[1])
+	if converr != nil {
+		fmt.Println("Not a number")
+		return
+	}
+
+	result, err := mayError(converted)
+
+	switch {
+	case len(os.Args) >= 3:
+		fmt.Println("Too many args")
+		return
+	case err != nil:
+		fmt.Println("Conversion failed")
+		return
+	}
+
+	fmt.Printf("Converted: %d\n", *result)
+
+}
+
+func definesValueInIf() {
+
+	converted, converr := strconv.Atoi(os.Args[1])
+	if converr != nil {
+		fmt.Println("Not a number")
+		return
+	}
+
+	var result *int
+	var err error
+	if result, err = mayError(converted); err != nil {
+		return
+	}
+
+	fmt.Printf("Converted: %d\n", *result)
```

Comment:
```suggestion
	fmt.Printf("Converted: %d\n", *result) // OK
```

and similar elsewhere